public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -227,5 +225,9 @@ export function runSplitQuery(datasource: LokiDatasource, request: DataQueryRequ
     });
   }
 
-  return runSplitGroupedQueries(datasource, requests);
+  const startTime = new Date();
+  return runSplitGroupedQueries(datasource, requests).pipe(
+    last(),
```

Comment:
Can I ask why was this `last` added? Is it related to tracking?

Reply:
The `last` here is breaking split functionality. The graph is not loading partial results and only shows results when they are all in. 

https://github.com/grafana/grafana/assets/30407135/98c81190-5ff8-42f0-b729-77cb60992fd9